control-station/package.json

```diff
@@ -12,6 +12,10 @@
 		"preview": "vite preview"
 	},
 	"dependencies": {
+		"@emotion/react": "^11.11.4",
+		"@emotion/styled": "^11.11.5",
+		"@mui/material": "^5.15.19",
```

Comment:
Question: are these other libraries needed when we're only using x-charts?

Reply:
These libraries are required with x-charts